zaza/openstack/charm_tests/manila/tests.py

```diff
@@ -85,6 +86,29 @@ def test_manila_api(self):
     def _list_shares(self):
         return self.manila_client.shares.list()
 
+    def test_902_nrpe_service_checks(self):
+        """Confirm that the NRPE service check files are created."""
+        units = zaza.model.get_units('manila')
+        services = ['apache2', 'haproxy', 'manila-scheduler', 'manila-data']
+
+        cmds = []
+        for check_name in services:
+            cmds.append(
+                'egrep -oh /usr/local.* /etc/nagios/nrpe.d/'
+                'check_{}.cfg'.format(check_name)
+            )
+
+        for attempt in tenacity.Retrying(
+            retry=tenacity.retry_if_result(lambda ret: ret is not None),
+            wait=tenacity.wait_fixed(120),
+            stop=tenacity.stop_after_attempt(2)
+        ):
+            with attempt:
+                ret = generic_utils.check_commands_on_units(cmds, units)
+                if ret:
+                    logging.info(ret)
```

Comment:
There is no need to log `ret` here as this will be logged already thanks to the `assertIsNone`